lib/cmd/fh3/undeploy.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+module.exports = undeploy;
+undeploy.undeploy = undeploy;
+
+undeploy.desc = i18n._("Undeploy a cloud app");
+undeploy.usage = "fhc undeploy <app-id> [apptype] --env=<environment>";
+
+var log = require("../../utils/log");
+var fhc = require("../../fhc");
+var fhreq = require("../../utils/request");
+var common = require("../../common");
+var ini = require('../../utils/ini');
+
+// Main undeploy entry point
+function undeploy(argv, cb) {
```

Comment:
Format of the command looks wrong. Did you copy that from fh2 commands?
For new fh3 commands we support different/object style commands.
I think that one of the team members can help you with migration or take over this PR.